src/main/kotlin/eu/europa/ec/eudi/openid4vci/Issuance.kt

```diff
@@ -248,6 +249,21 @@ interface RequestIssuance {
         requestPayload: IssuanceRequestPayload,
     ): Result<SubmittedRequest>
 
+    @Deprecated(
```

Comment:
Since we introduce "breaking" changes in config and batch credential request lets follow the same for the single request issuance.